.gitignore

```diff
@@ -0,0 +1,7 @@
+smtprelay
+smtprelay.log
+*.ini
+!smtprelay.ini
+# Ignore https://github.com/cosmtrek/air config file
+.air.toml
```

Comment:
PS you can add a global gitignore - this helps keep the repo's own `.gitignore` file clean